sdmetrics/visualization.py

```diff
@@ -365,6 +365,28 @@ def _generate_column_plot(
     return fig
 
 
+def _get_max_between_datasets(real_data, synthetic_data):
+    if synthetic_data is None and real_data is None:
+        raise ValueError('Cannot get max between two None values.')
+    if real_data is None:
+        return max(synthetic_data)
+    elif synthetic_data is None:
+        return max(real_data)
+    else:
+        return max(max(real_data), max(synthetic_data))
+
+
+def _get_min_between_datasets(real_data, synthetic_data):
```

Comment:
can we add unit tests for these methods or unit tests for `_generate_cardinality_plot` that will test these code paths

Reply:
Added